src/PhpSpreadsheet/Reader/Xml.php

```diff
@@ -138,6 +138,9 @@ public function listWorksheetNames($pFilename)
             'SimpleXMLElement',
             Settings::getLibXmlLoaderOptions()
         );
+        if (!($xml instanceof \SimpleXMLElement)) {
+            throw new Exception('SimpleXMLElement can not load ' . $pFilename);
```

Comment:
Should compare against `false` instead of checking types, so `if ($xml === false)`.

Error message should include error details from simplexml.

Reply:
function **simplexml_load_string** returns:
>SimpleXMLElement an object of class SimpleXMLElement with * properties containing the data held within the xml document, or <b>FALSE</b> on failure.

It just return **false** when error occurred, so there's no more error information.